Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/TabbedPageTests.cs

```diff
@@ -21,6 +21,9 @@ namespace Xamarin.Forms.Controls.Issues
 	[Issue (IssueTracker.None, 0,"TabbedPage nav tests", PlatformAffected.All)]
 	public class TabbedPageTests : TestTabbedPage
 	{
+		const string _page1AutomationId = "Page 1";
```

Comment:
Make these different from the tab titles so we're sure it's automating on the right thing.

```suggestion
		const string _page1AutomationId = "Auto 1";
```